com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKey.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UX
+{
+    /// <summary>
+    /// Abstract class representing a key in the non native keyboard
+    /// </summary>
+    public abstract class NonNativeKey : MonoBehaviour
```

Comment:
Does this need to be public?

Reply:
In case others want to create their own fancy buttons? Don't have strong opinion on it though